app/presenters/content_item_presenter.rb

```diff
@@ -1,14 +1,15 @@
 class ContentItemPresenter
   include Withdrawable
 
-  attr_reader :content_item, :base_path, :title, :description, :format, :locale, :phase, :document_type
+  attr_reader :content_item, :base_path, :title, :description, :format, :locale, :phase, :document_type, :schema_name
 
   def initialize(content_item)
     @content_item = content_item
     @base_path = content_item["base_path"]
     @title = content_item["title"]
     @description = content_item["description"]
     @format = content_item["schema_name"]
+    @schema_name = content_item["schema_name"]
```

Comment:
Could you instead refactor out the deprecated use of format, ie the line above this?
Note: `@format = content_item["schema_name"]`